website_sale_cart_preview/static/src/css/website_sale_cart_preview.css

```diff
@@ -0,0 +1,26 @@
+.mycart-popover .cart_line {
```

Comment:
About this, I was looking at the OCA builds on runbot and you are right, it is necessary to install website_less module to compile .less files, and is not available, so nevermind.